### DISCUSSION TITLE
Show only online chats in tabs

### DISCUSSION THREAD

**Discussion by scrimitzu** [Chatterino/chatterino2#4353] [Ideas] [answered]
Pretty straightforward, an option to show only tabs/chats for streams that are currently online, if possible. I think it would be a great feature.

**Comment by Felanbird** [comment]
Was being worked on in #3658 but it's unfortunately stale

**Comment by pajlada** [comment, marked as answer]
The latest nightly version has this this functionality where you can configure Chatterino to only show tabs where one of the chats in it has a stream that is online.

Implemented in https://github.com/Chatterino/chatterino2/pull/4358

**Comment by scrimitzu** [comment]
Omg thank you so much!!!
